packages/surfey/Classes/Controller/SurveyController.php

```diff
@@ -0,0 +1,32 @@
+<?php
+namespace TYPO3Incubator\Surfey\Controller;
+
+use TYPO3\CMS\Core\Utility\GeneralUtility;
+use TYPO3\CMS\Fluid\View\StandaloneView;
+
+class SurveyController
+{
+    /**
+     * Renders the survey content element
+     *
+     * @param string $content The content
+     * @param array $conf The TypoScript configuration
+     * @return string The rendered content
+     */
+    public function render($content, $conf): string
+    {
+        $view = GeneralUtility::makeInstance(StandaloneView::class);
```

Comment:
Let's use the ViewFactoryInterface to create the view, see e.g. XmlSitemapRenderer as an example.